service/insolvency_service.go

```diff
@@ -242,6 +242,15 @@ func ValidateAntivirus(svc dao.Service, insolvencyResource models.InsolvencyReso
 
 	validationErrors := make([]models.ValidationErrorResponseResource, 0)
 
+	// Check if transactionID is empty to check if insolvency resource was found in DB
+	if insolvencyResource.TransactionID == "" {
+		log.Info("insolvency case not found")
```

Comment:
I'd add to the log what the transaction ID is for the not-found case otherwise it's not too helpful for people checking logs